http/src/request/application/mod.rs

```diff
@@ -1,60 +1,10 @@
-pub mod create_followup_message;
+pub mod command;
+pub mod interaction;
 
-mod create_global_command;
-mod create_guild_command;
-mod delete_followup_message;
-mod delete_global_command;
-mod delete_guild_command;
-mod delete_original_response;
-mod get_command_permissions;
-mod get_global_commands;
-mod get_guild_command_permissions;
-mod get_guild_commands;
-mod get_original_response;
-mod interaction_callback;
-mod set_command_permissions;
-mod set_global_commands;
-mod set_guild_commands;
-mod update_command_permissions;
-mod update_followup_message;
-mod update_global_command;
-mod update_guild_command;
-mod update_original_response;
-
-pub use self::{
```

Comment:
For this to go into `main` and not be breaking we have to continue re-exporting from here.